specification/monitor/resource-manager/Microsoft.Insights/stable/2021-09-01/actionGroups_API.json

```diff
@@ -242,9 +242,137 @@
         ],
         "responses": {
           "202": {
+            "description": "The notification request accepted",
+            "headers": {
+              "location": {
+                "type": "string",
+                "description": "The location header that has the polling uri."
+              }
+            }
+          },
+          "200": {
+            "description": "The notification succeeded",
+            "schema": {
+              "$ref": "#/definitions/TestNotificationDetailsResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while sending the test notifications",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/createNotifications": {
```

Comment:
Why we are only adding the new operations in this api-version instead of the newer ones? Is it because they already have these?

Reply:
> Why we are only adding the new operations in this api-version instead of the newer ones? Is it because they already have these?

The newer version already had these changes. I just bring whatever we had in the new versions back to this one as in the backend, the APIs work the same.